adapters/adnuntius/adnuntius.go

```diff
@@ -109,13 +121,41 @@ func makeEndpointUrl(ortbRequest openrtb2.BidRequest, a *adapter) (string, []err
 		q.Set("gdpr", gdpr)
 		q.Set("consentString", consent)
 	}
+
+	if deviceExt.NoCookies {
+		q.Set("noCookies", "true")
+	}
+
 	q.Set("tzo", fmt.Sprint(tzo))
 	q.Set("format", "json")
 
 	url := a.endpoint + "?" + q.Encode()
 	return url, nil
 }
 
+func getImpSizes(imp openrtb2.Imp) [][]int64 {
+	if imp.Banner == nil {
+		return nil
```

Comment:
Could you add a test where imp.Banner is nil?

Reply:
Added test for this! :)

Reply:
Thank you for adding tests. 
This part of code is still not covered even after you added a test with impression without banner. Test exits on Line 175, there is another check if banner is nil.  Can you remove this line here? It's redundant.

Reply:
Myt bad, should be fixed with the latest pull request.